instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/DatabaseClientTracer.java

```diff
@@ -18,7 +18,7 @@
 import java.net.InetSocketAddress;
 import java.util.concurrent.ExecutionException;
 
-public abstract class DatabaseClientTracer<CONNECTION, QUERY> extends BaseTracer {
+public abstract class DatabaseClientTracer<ConnectionT, QueryT> extends BaseTracer {
```

Comment:
Just a random pick: not sure I like this change :(

Reply:
I reverted the type variable names, and will submit a separate PR to discuss those changes (I had done that change as a separate commit suspecting potential objection 😁)